JSQMessagesViewController/Factories/JSQMessagesToolbarButtonFactory.m

```diff
@@ -22,10 +22,27 @@
 #import "UIImage+JSQMessages.h"
 #import "NSBundle+JSQMessages.h"
 
+@interface JSQMessagesToolbarButtonFactory ()
+
+@property (strong, nonatomic, readonly) UIFont *buttonFont;
+
+@end
 
 @implementation JSQMessagesToolbarButtonFactory
 
-+ (UIButton *)defaultAccessoryButtonItem
+- (instancetype)initWithFont:(UIFont *)font
+{
+    NSParameterAssert(font != nil);
+    
+    self = [super init];
+    if (self) {
+        _buttonFont = font;
+    }
+    
+    return self;
+}
+
```

Comment:
we should implement `init` and call this initializer with our default font, `[UIFont boldSystemFontOfSize:17.0]`

```objc
- (instancetype)init {
    return [self initWithFont:[UIFont boldSystemFontOfSize:17.0]];
}
```